front/src/styles/scss/common/components/_osrdMenu.scss

```
@@ -0,0 +1,45 @@
+.osrd-menu {
+  position: absolute;
+  left: -0.1875rem;
+  z-index: 10;
+  background-color: var(--grey5);
+  outline: 0.0625rem solid var(--grey30);
+  border-radius: 0.5rem;
+  box-shadow:
+    0px 6px 21px -5px rgba(101, 169, 232, 0.341810533216783),
+    0px 16px 30px -5px rgba(0, 0, 0, 0.2),
+    0px 3px 5px -2px rgba(0, 0, 0, 0.1);
```

```suggestion
    0 0.375rem 1.3125rem -0.3125rem rgba(101, 169, 232, 0.341810533216783),
    0 1rem 1.875rem -0.3125rem rgba(0, 0, 0, 0.2),
    0 0.1875rem 0.3125rem -0.125rem rgba(0, 0, 0, 0.1);
```

are you sure about the alpha `0.341810533216783` ?

---

Good point for the rems. All this box-shadow comes from the mockup so I just kept the original values.